Armor Up! Survivor Expansion/modinfo.json

```diff
@@ -4,8 +4,9 @@
     "ident": "ArmUpSurvEx",
     "name": "Armor Up! Survivor Expansion",
     "authors": "darktoes",
-    "description": "A few extra bits of Survivor-grade armor.",
+    "maintainers": "DanteTorn",
+    "description": "A few extra bits of Survivor-grade armor. -Updated for 1 second turns.-",
```

Comment:
Maybe add a changelog.txt and just put a date of update here.

Reply:
Done.